tests/dbt/test_adapter.py

```diff
@@ -58,6 +63,76 @@ def test_adapter_relation(sushi_test_project: Project, runtime_renderer: t.Calla
         == "[]"
     )
 
+    adapter_mock = mocker.MagicMock()
+    adapter_mock.dialect = "snowflake"
+
+    context.target = SnowflakeConfig(
+        account="test",
+        user="test",
+        authenticator="test",
+        name="test",
+        database="test",
+        schema="test",
+    )
+
+    renderer = runtime_renderer(context, engine_adapter=adapter_mock)
+
+    # bla and bob will be normalized to uppercase since we're dealing with Snowflake
+    bla_id = exp.to_identifier("BLA", quoted=False)
+    bob_id = exp.to_identifier("BOB", quoted=False)
+
+    renderer("{{ adapter.get_relation(database=None, schema='bla', identifier='bob') }}")
+    adapter_mock._get_data_objects.assert_has_calls([call(exp.Table(db=bla_id))])
+
+    renderer(
+        "{%- set relation = api.Relation.create(schema='bla') -%}"
+        "{{ adapter.create_schema(relation) }}"
+    )
+    adapter_mock.create_schema.assert_has_calls([call(bla_id)])
+
+    renderer(
+        "{%- set relation = api.Relation.create(schema='bla') -%}"
+        "{{ adapter.drop_schema(relation) }}"
+    )
+    adapter_mock.drop_schema.assert_has_calls([call(bla_id)])
```

Comment:
Why twice with the same params?

Reply:
One should be for `create_schema`, the other for `drop_schema`